card/mod/machines/lib/javascript/wagn.js.coffee

```diff
@@ -385,3 +385,7 @@ snakeCase = (str)->
               match[1].toLowerCase()
 
 warn = (stuff) -> console.log stuff if console?
+
+wagn.slotReady (slot) ->
+  slot.find('._disappear').delay(5000).animate(
```

Comment:
What are the alternatives to CSS classes? @srivig started the practice on wikirate to prefix CSS classes with an underscore if they are javascript signifiers. That seems to me like a reasonable practice.